Assets/Scripts/CommonComponents/EndGame.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class EndGame : MonoBehaviour
+{
+    public HealthManager player;
+    public void Start()
+    {
+     	HealthManager player = GameObject.FindWithTag("Player").GetComponent<HealthManager>();   
```

Comment:
Why didn't you use GetComponentInChildren since we are putting this script on the player anyway?

Reply:
cause i don't want to